riff-raff.yaml

```diff
@@ -0,0 +1,21 @@
+stacks:
+- discussion
+deployments:
+  static:
+    parameters:
+      cacheControl:
+      - pattern: .*\.js$
+        value: public, max-age=315360000
+      - pattern: .*\.map$
+        value: public, max-age=315360000
+      - pattern: .*
+        value: public, max-age=60
+      mimeTypes:
+        js: application/javascript
+        json: application/json
+        map: text/plain
+      prefixStack: false
```

Comment:
@sihil how do you feel about this for cases like the s3-upload type where the default (for publicACL) is potentially unsafe? We could change the default but it would break backwards compatibility, but alternatively I would prefer to include it in the RiffRaff file without getting the scary warning.